### ISSUE TITLE
SynopsysTool and MentorTool belong in their respective cad-plugins repos

### ISSUE THREAD

**Issue by jwright6323** [ucb-bar/hammer#185]
It probably makes sense to include the CadenceTool and SynopsysTool (and future InsertVendorTool) classes with their subclasses.

https://github.com/ucb-bar/hammer/blob/b8ef3a9a12275a77df7a989e6728771112e5f531/src/hammer-vlsi/hammer_vlsi/hammer_vlsi_impl.py#L370

**Comment by edwardcwang** [comment]
Related to #198

**Comment by harrisonliew** [comment]
CadenceTool moved by #576 but SynopsysTool & MentorTool remain

**Comment by harrisonliew** [comment]
There needs to be a better mechanism for the respective plugin repos to import their common code. Currently, for cadence tools, we have to do this:

```
sys.path.append(os.path.join(os.path.dirname(os.path.realpath(__file__)),"../../common"))
from tool import CadenceTool
```

**Comment by harrisonliew** [comment]
Solved in #713 and packaging infrastructure.
